xml/templates/dao.tpl

```diff
@@ -30,7 +30,7 @@ class {$table.className} extends CRM_Core_DAO {ldelim}
 {foreach from=$table.fields item=field}
     /**
 {if $field.comment}
-     * {$field.comment}
+     * {$field.comment|replace:"\n":"\n* "}
```

Comment:
a regex replace that also removed the extra whitespace would be super cool. Would this work?

    |preg_replace:"/\n[ ]*/":"\n* "

Reply:
@colemanw that gives me `Delimiter must not be alphanumeric or backslash`

Reply:
Whoops, looks like it should be `|regex_replace`
https://www.smarty.net/docsv2/en/language.modifier.regex.replace.tpl

Reply:
That seems to have worked @colemanw